HEN_HOUSE/src/EEMF_macros.mortran

```diff
@@ -0,0 +1,2397 @@
+REPLACE{$MINS_ANG}WITH{0.05};
+
+"Default delta_u value if input file definitions are missing"
+REPLACE{$deltaUDefault}WITH{0.02};
+
+"Maximum energy loss due to electric field (step-size restriction)"
+REPLACE{$MAXS_eLoss}WITH{0.1};
+
+" ******** algorithm options  ******** "
+"Integration order selection for PRESTA"
+"   Value can be 1, 2 or 3"
+REPLACE{$IntegrationAlg}WITH{3};
+"Single scattering algorithm selection"
+" Value can either be 1 or 2:"
+" 1: A 1 point integration of the effect of the EM field"
+" 2: Analytical solution to propogration of particle in magnetic field."
+"    (defaults back to 1 point integration for electric fields "
+REPLACE{$ssFieldAlg}WITH{2};
+
+"******** BCA variables ******** "
+" If using BCA set these parameter "
+REPLACE{$BCA_lin_buffer}WITH{min(1e-6,skindepth)};
+REPLACE{$BCA_lin_dist}WITH{min(1e-5,skindepth)};
+
+
+" ****************** BOOLEANS *****************"
+REPLACE{$orderBoundary}WITH{.true.}; "selecting 1PI v 3PI so no boundary errors"
+REPLACE{$orderAdaptive}WITH{.true.}; "Selecting 1PI v 3PI for B for efficiency"
+REPLACE{$duAdaptive}WITH{.true.}; "for B, scaling the step size to a 1.5 T"
+REPLACE{$energyLossViaPotential}WITH{.false.};"Boolean for using potential"
+      "for energy loss. "
+" Determine if output at the begining of the simulation is needed: "
+REPLACE{$EMInternalOutput}WITH{.true.};
+
+
+"******************************** CONSTANTS ***********************************"
+REPLACE{$EM_MACROS_ACTIVE}WITH{.true.};
+REPLACE{$SL}WITH{299792458.0}; "speed of light"
+REPLACE{$ERM}WITH{0.510998910}; "electron rest energy in MeV"
+REPLACE{$PI_em}WITH{3.14159265}; "Pi"
+REPLACE{$ERM_kg}WITH{9.10938356E-31}; "electron mass in kg"
+REPLACE{$EC_em}WITH{1.60217662E-19}; "electron charge"
+"parameter for rounding errors"
+REPLACE{$epsilon_em}WITH{1e-14};
+"**************************** changing fields *********************************"
+" Changing magnetic and electric fields during a step has not been implemented "
+" yet. These variables are for some of the initial set up of the algorithm."
+REPLACE{$changingEMStepDist}WITH{1e-3};
+REPLACE{$changingEMField}WITH{.false.};
+" The following allows turning off single scateer everywhere except near       "
+" boundaries. This requires an adjustment to the egsnrc.mortran code to work   "
+" and should be left as .true. unless properly implementing this option        "
+" This is left in for ease of use for testing. "
+REPLACE{$ss_em}WITH{.true.};
+"******************************************************************************"
+"******************************************************************************"
+
+"*****************************************************************************"
+"*****************************************************************************"
+" *****         COMMON block for EM field         *****"
+REPLACE{;COMIN/EM/;}WITH{;COMMON/EM/
+    " Variables for the definition of the magnetic field: "
+    B_amp, Bx, By, Bz,
```

Comment:
Currently this won't compile. This is because egsnrc.macros now includes `COMIN/EMF-INPUTS` which has already defined the Bx, By and Bz variables. Rename them in this file to Bxx, Byy and Bzz and it compiles and the magnetic fields work.